src/_utils.ts

```diff
@@ -176,7 +178,7 @@ export function hasEncryptedConnection(req) {
  * @api private
  */
 
-export function urlJoin(...args: string[]) {
+export function urlJoin(options: ProxyServerOptions, ...args: string[]) {
```

Comment:
I think we can alternatively use `join` from `ufo`. It handles join which much better care.

Reply:
Done 💪🏻